kivy/properties.pyx

```diff
@@ -1438,15 +1442,11 @@ cdef class AliasProperty(Property):
         `getter`: function
             Function to use as a property getter.
         `setter`: function
-            Function to use as a property setter. Callbacks bound to the
-            alias property won't be called when the property is set (e.g.
-            `right = 10`), unless the setter returns `True`.
+            Function to use as a property setter.
```

Comment:
I'm not sure why you removed this?

Reply:
Removed sentence is partially true, because callbacks will be also dispatched when some of binded properties changes and getter returns new value. 
I thought that it would be better to explain in text above how setter should work and don't repeat same thing in `Parameters` section.

Reply:
I think it is better to give a very short summary here, even if it was already explained earlier because sometimes I don't get the full implication from the example and I want a short summary to make sure I understood the implications. Repeating it twice gives me chance to make sure I understood by comparing what I understand the two places to say.

So I'd say it's good to fix it to give a very short and concise summary here and for `bind`.